app/build.gradle

```diff
@@ -60,34 +63,6 @@ dependencies {
     implementation composeBom
     androidTestImplementation composeBom
 
```

Comment:
Sorry, it's not clear to me, why are you removing these dependencies? Is it because we're switching from the rooms and db to hilt injection?

Reply:
They were removed by accident. Adding them back. Thanks for the pointer.